### ISSUE TITLE
using `yolov9_custom.yaml` with `yolov9-e.pt` weights

### ISSUE THREAD

**Issue by KoniHD** [AarohiSingla/YOLOv9#4]
Hi @AarohiSingla 

first of all thank you for the amazing YouTube video for working on custom dataset! It definetely helped me getting my thesis of the ground 😃 

I just want point out a major problem with your train_dual.py commad which is very unfortunate. For training on custom data you are using a modified version of [yolov9.yaml](https://github.com/WongKinYiu/yolov9/blob/main/models/detect/yolov9.yaml) in combination with `yolov9-e.pt` weights, thereby creating a mismatch. You can see this in the output below the training cell where only 438/1412 items from the `yolov9-e.pt` weights are transferred and consequently used.

> 38[31, 34, 37, 16, 19, 22]  1  21547442  models.yolo.DualDDetect                 [3, [512, 512, 512, 256, 512, 512]]
>yolov9_custom summary: 930 layers, 60801842 parameters, 60801810 gradients, 266.1 GFLOPs
>
>Transferred 438/1412 items from /mydrive/yolov9/yolov9-e.pt
>AMP: checks passed ✅

The effect of this is that the initial mAP in epoch 0 is 0.0153 even though pre-trained weights are used. Another effect is that you only need ~5.41GB VRAM.

Now in my opinion for a proper use of the `yolov9-e.pt` weights a modified version of [yolov9-e.yaml](https://github.com/WongKinYiu/yolov9/blob/main/models/detect/yolov9-e.yaml) should be used instead. When I found this error in your instruction I fixed it which caused my model to roughly get a mAP of 0.3 in epoch 0 while using ~20GB VRAM.

However, I might be wrong here. In that case please excuse me but I also asked the original creator of the YOLOv9 model which configuration the [yolov9.yaml](https://github.com/WongKinYiu/yolov9/blob/main/models/detect/yolov9.yaml) file is closest to but to me it doesn't seem to be YOLOv9-E (see [#536](https://github.com/WongKinYiu/yolov9/issues/536)).
